app/datatables/parent_object_datatable.rb

```diff
@@ -100,6 +100,10 @@ def extent_of_full_text(parent_object)
     "Yes"
   end
 
+  def metadata_sources
+    ["ladybird", "aspace", "ils", "sierra"]
+  end
+
```

Comment:
This could move to MetadataSource so it is only in one place, and pull from the database.  Something like:
```
def self.all_metadata_cloud_names
  MetadataSource.distinct.pluck(:metadata_cloud_name)
end 
```